lib/egg.js

```diff
@@ -40,7 +40,7 @@ class EggApplication extends EggCore {
      */
     this.messenger = new Messenger();
 
-    this.dumpConfig();
+    this.beforeStart(() => this.dumpConfig());
```

Comment:
but sometimes we want to check the config file to figure out why server is not ready